dipy/utils/parallel.py

```diff
@@ -50,6 +50,13 @@ def paramap(func, in_list, out_shape=None, n_jobs=-1, engine="joblib",
     ndarray of identical shape to `arr`
 
     """
+
+    if func_args is None:
+        func_args = []
+
+    if func_kwargs is None:
+        func_kwargs = {}
```

Comment:
`func_args = func_args or []` instead

`func_kwargs = func_kwargs or {}` instead

Reply:
Done in f49517e.